public/stylesheets/landing.css

```diff
@@ -0,0 +1,43 @@
+canvas {
```

Comment:
if you are **more** specific than bootstrap, your style will override it. Try something like